javascript.js

```diff
@@ -100,4 +100,28 @@ for (let m = 0; m < seeProject.length; m += 1) {
   seeProject[m].addEventListener('click', display2);
 }
 
-xIcon.addEventListener('click', display2);
\ No newline at end of file
+xIcon.addEventListener('click', display2);
+
+const form = document.getElementById('form');
+const small = document.getElementsByTagName('small')[0];
+const email = document.getElementById('email');
+
+function validateEmail(input) {
+  const address = input.toString();
+  let isLower = false;
+  if (address === address.toLowerCase()) {
+    isLower = true;
+  } else {
+    isLower = false;
+  }
+  return isLower;
+}
```

Comment:
- [ ] Kindly when I enter an email without capital letters it shows me the error message. Kindly fix it.

![image](https://user-images.githubusercontent.com/35535130/156486193-592d48da-2f76-4586-b827-2e7e4abee809.png)